src/Core/src/Platform/Android/TransformationExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AView = Android.Views.View;
+
+namespace Microsoft.Maui
+{
+	public static class TransformationExtensions
+	{
+		public static void UpdateTranslationX(this AView nativeView, IView view)
+		{
+			if (nativeView.Context == null)
+				return;
+
+			nativeView.TranslationX = nativeView.Context.ToPixels(view.TranslationX);
+		}
+
+		public static void UpdateTranslationY(this AView nativeView, IView view)
+		{
+			if (nativeView.Context == null)
+				return;
+
+			nativeView.TranslationY = nativeView.Context.ToPixels(view.TranslationY);
+		}
+
+		public static void UpdateScale(this AView nativeView, IView view)
+		{
+			nativeView.UpdateScaleX(view);
+			nativeView.UpdateScaleY(view);
+		}
+
+		public static void UpdateScaleX(this AView nativeView, IView view)
+		{
+			nativeView.ScaleX = (float)view.Scale * (float)view.ScaleX;
+		}
+
+		public static void UpdateScaleY(this AView nativeView, IView view)
+		{
+			nativeView.ScaleY = (float)view.Scale * (float)view.ScaleY;
+		}
+
+		public static void UpdateRotation(this AView nativeView, IView view)
+		{
+			nativeView.Rotation = (float)view.Rotation;
+		}
+
+		public static void UpdateRotationX(this AView nativeView, IView view)
+		{
+			nativeView.RotationX = (float)view.RotationX;
+		}
+
+		public static void UpdateRotationY(this AView nativeView, IView view)
+		{
+			nativeView.RotationY = (float)view.RotationY;
+		}
+
+		public static void UpdateAnchorX(this AView nativeView, IView view)
+		{
+			if (nativeView.Context == null)
+				return;
+
+			float currentPivot = nativeView.PivotX;
+			var target = (float)(view.AnchorX * nativeView.Context.ToPixels(view.Width));
```

Comment:
Does this mean that this needs to be called again when the width changes?

Reply:
Good question. Also, I see this is using `view.Width` and iOS is using `view.Frame.Width`. I think iOS is more right as that is supposed to be the "actual" bounds.

Reply:
I don't think there is an equivalent to ```.Frame``` on Android. There is just ```.Width``` and ```.Height``` but Idk maybe there is.

Basically, this needs to be called every OnLayout/Arrange.

Reply:
Yes, need to be called again if View Frame changes.

Reply:
This is on the virtual view. I think the Width/Height is the user values and Frame is the actual size.

Reply:
Apply changes refresh transformations when size changes.